oss/chromium/base/numerics/checked_math_impl.h

```diff
@@ -266,7 +266,7 @@ struct CheckedLshOp {};
 
 // Left shift. Shifts less than 0 or greater than or equal to the number
 // of bits in the promoted type are undefined. Shifts of negative values
-// are undefined. Otherwise it is defined when the result fits.
+// are undefined. Otherwise, it is defined when the result fits.
```

Comment:
This is an external dependency so we should probably exclude it and the rest of the `oss` directory. What do you think?

Reply:
Sure.

Reply:
Oops, it's actually excluded by the rules, I just was using VSCode to apply the fix instead of using my normal tooling ☺️.